audiorecorder/src/main/java/org/odk/collect/audiorecorder/recording/AudioRecorder.kt

```diff
@@ -1,16 +1,15 @@
 package org.odk.collect.audiorecorder.recording
 
 import androidx.lifecycle.LiveData
-import androidx.lifecycle.ViewModel
 import org.odk.collect.audiorecorder.recorder.Output
 import java.io.File
 import java.io.Serializable
 
 /**
- * Interface for a ViewModel that records audio. Can only record once session
+ * Interface for a component that records audio. Can only record once session
  * at a time.
  */
-abstract class AudioRecorderViewModel : ViewModel() {
+abstract class AudioRecorder {
```

Comment:
I'm getting a stronger and stronger feeling that we're going to want to wrap this in a `ViewModel` on the Collect side. I also keep confusing myself with lifecycle as this component's state is really at the singleton level and not related to the view/activity lifecycle.